code/__HELPERS/_lists.dm

```diff
@@ -438,6 +438,37 @@
 			L1[key] = other_value
 
 
+/// Turns an associative list into a flat list of keys
+/proc/assoc_to_keys(list/input)
+	var/list/keys = list()
+	for(var/key in input)
+		keys += key
+	return keys
+
+/proc/compare_list(list/l,list/d)
```

Comment:
a comment on what this does for clarity please. Because the name is shit.

Reply:
done